03.asynchronous/callback.js

```diff
@@ -0,0 +1,36 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory");
+
+const executeWithoutError = () =>
+  db.run(
+    "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE )",
```

Comment:
余分なスペースが含まれています。他にも同じような箇所がないか確認して修正してください。

```suggestion
    "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
```

Reply:
修正しました!

[余分な空白を削除しました](https://github.com/tyrrell-IH/js-practices/pull/3/commits/456f40b1b672115152c19b85d27913c49a0f0893)